api/v1alpha1/sku_handler/cloud_sku_handler.go

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package skuhandler
+
+type CloudSKUHandler interface {
+	GetSupportedSKUs() string
+	IsSupportedSKU(sku string) (GPUConfig, bool)
```

Comment:
Why return GPUConfig?

Reply:
During validation, we need the GPUcount and GPUMem information: https://github.com/Azure/kaito/blob/6be8a0d57225f776d1fe01bd6069f7d3f894eff0/api/v1alpha1/workspace_validation.go#L247

Reply:
The inference method name "IsSupportedSKU" clearly indicates that the return type should be just a bool.

Reply:
I gave this some thought and changed the method to GetGPUConfigMap which just returns the map as is. Maybe we won't need the GetSupportedSKUs method after this but I think its better to keep it to keep the code clean